includes/api/class-edd-api-v2.php

```diff
@@ -368,11 +368,11 @@ public function get_recent_sales() {
 				$user_info    = $payment->user_info;
 
 				$sales['sales'][ $i ]['ID']             = $payment->number;
-				$sales['sales'][ $i ]['transaction_id'] = $payment->transaction_id;
+				$sales['sales'][ $i ]['transaction_id'] = ( ! empty( $payment->transaction_id ) ) ? $payment->transaction_id : null;
```

Comment:
shouldn't this be:

`( ! empty( $payment->transaction_id ) ? $payment->transaction_id : null);`